ThreeDSecure/src/main/java/com/braintreepayments/api/CardinalResult.java

```diff
@@ -0,0 +1,42 @@
+package com.braintreepayments.api;
+
+import com.cardinalcommerce.cardinalmobilesdk.models.ValidateResponse;
+
+class CardinalResult {
```

Comment:
This class is needed to support a single result object required by the `ActivityResultContract`